assets/js/script.js

```diff
@@ -1,21 +1,18 @@
+import navLinks from './modules/nav.js';
```

Comment:
- [ ] @Mukaba kindly change the name of this file from script to index.js file as per the project instructions
![image](https://user-images.githubusercontent.com/61408860/228283372-4f759e9d-d827-4c50-a1b8-024e6d854628.png)

- [ ] [optional] It would be better if from moved your js files from the assets folder, it's common practice to only have images and styling stored in the assets directory.